### PR TITLE
Add alt attribute to the gallery image

### LINKED ISSUE #96: Add `alt` for the `img` elements

### PR DESCRIPTION
**Pull request by feimosi**
Resolves #96

### CODE REVIEW

**Comment by feimosi** [comment]
@XhmikosR I'd appreciate if you could take a look and approve this PR :)

File: src/baguetteBox.js [hunk 1]
```diff
@@ -491,21 +491,25 @@
                             options.captions.call(currentGallery, imageElement) :
                             imageElement.getAttribute('data-caption') || imageElement.title;
         var imageSrc = getImageSrc(imageElement);
-        // Prepare image container elements
-        var figure = create('figure');
-        var image = create('img');
-        var figcaption = create('figcaption');
 
+        // Prepare figure and figcaption elements
```
Comment (XhmikosR) [inline src/baguetteBox.js]: "and figcaption elements" belongs to line 504 now I believe.

**Comment by XhmikosR** [comment]
Looks good to me sans the refactoring patch which I assume you tested :)

File: src/baguetteBox.js (hunk 1 above)
Comment (feimosi) [inline src/baguetteBox.js]: Right, I've removed this unnecessary keyword

**Comment by feimosi** [comment]
I wouldn't publish untested code :) Just wanted to make sure, we were thinking about the `alt` in the same way.

**Comment by XhmikosR** [comment]
I know, better be safe than sorry :D

**Pull request merged**